### PR TITLE
Add config instruction for archival node

### PR DESCRIPTION
**Pull request by janewang**
* Consolidating instructions to run an archival node under Run a Node doc

### CODE REVIEW

File: docs/develop/node/archival/run-archival-node.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+id: run-archival-node
+title: Run an Archival Node
+sidebar_label: Run an Archival Node
+description: Run an Archival Node
+---
+
+Running an archival node is the same as a [validator node](/docs/develop/node/validator/running-a-node) as both types of node use the same `nearcore` release. The key difference for running an archival node is a modification to the `config.json` by changing `archive` to `true` and specifying `tracked_shards`.
```
Comment (bowenwang1996) [inline docs/develop/node/archival/run-archival-node.md:8]: Right now `tracked_shards` is not different for other types of nodes.

File: docs/develop/node/archival/run-archival-node.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+| Network | URL                                                                                         |
+| ------- | ------------------------------------------------------------------------------------------- |
+| Mainnet | https://near-protocol-public.s3.ca-central-1.amazonaws.com/backups/mainnet/archive/data.tar |
+| Testnet | https://near-protocol-public.s3.ca-central-1.amazonaws.com/backups/testnet/archive/data.tar |
+
+---
+
+## Steps to Run an Archival Node
+
+Make sure [`nearup`](https://github.com/near/nearup) is installed. You can install `nearup` by following the instructions at https://github.com/near/nearup.
```
Comment (bowenwang1996) [inline docs/develop/node/archival/run-archival-node.md:36]: We should add instructions on how to run an archival node without nearup as well.

**Review by frol — APPROVED** [review]

File: docs/develop/node/archival/hardware-archival.md [hunk 3]
```diff
@@ -53,3 +53,6 @@ All prices reflect *reserved instances* which offer deep discounts on all platfo
 - Azure — https://azure.microsoft.com/en-us/pricing/calculator
 
 </blockquote>
+
+> Got a question?
+> <a href="https://stackoverflow.com/questions/tagged/nearprotocol"> > <h8>Ask it on StackOverflow!</h8></a>
```
Comment (frol) [inline docs/develop/node/archival/hardware-archival.md:58]: Weirdly, this renders extra `>` in the text after "Got a question?"

https://docs-pr-742.onrender.com/docs/develop/node/archival/hardware-archival

image

**Review by frol — APPROVED** [review]

File: docs/develop/node/archival/run-archival-node.md (hunk 2 above)
Comment (chefsale) [reply docs/develop/node/archival/run-archival-node.md:35]: Do we also added the tracked shards in the documentation for the archival nodes? Here is seems we only mention the archive: true flag?

**Review by chefsale — APPROVED** [review]
LGTM, just please take a look at the comment here: https://github.com/near/docs/pull/742/files#r649917885, as I believe we should also mention the tracked_shards config alongside the archive flag,

**Comment by janewang** [comment]
> LGTM, just please take a look at the comment here: https://github.com/near/docs/pull/742/files#r649917885, as I believe we should also mention the tracked_shards config alongside the archive flag,

Thanks updated to config.

For server costs, I recalculated the estimated costs based on hourly for the instance type. AWS ended up the most expensive, which seems to be usually the case. Please take a look. Does this look alright to you @chefsale?

https://github.com/near/docs/pull/742/commits/3f9a963a2b92e5df4b506b5b2acad6a725f8e128

File: docs/develop/node/archival/run-archival-node.md (hunk 2 above)
Comment (bowenwang1996) [reply docs/develop/node/archival/run-archival-node.md:36]: @janewang feel free to consult with @chefsale on this if you are not sure about how to run an archival node without nearup.

Comment (chefsale) [reply docs/develop/node/archival/run-archival-node.md:36]: @janewang feel free to ping me in chat or even schedule a meeting on running a neard standalone. I would really like to help on this, also provide maybe a guide on how to run it with systemd as well, as this is the best way to do it and that is how we manage all our current nodes on testnet and mainnet.

**Comment by chefsale** [comment]
> > LGTM, just please take a look at the comment here: https://github.com/near/docs/pull/742/files#r649917885, as I believe we should also mention the tracked_shards config alongside the archive flag,
> 
> Thanks, updated the config.json for archival.
> 
> For server costs, I recalculated the estimated costs based on hourly for the instance type. AWS ended up the most expensive, which seems to be usually the case. Please take a look [3f9a963](https://github.com/near/docs/commit/3f9a963a2b92e5df4b506b5b2acad6a725f8e128). Does this look alright to you @chefsale?

Yes, this looks reasonable to me. 

I believe the SSDs are a bit more tricker, as there are two types of SSDs on GCP for example. But 300 dollars is a good rough estimate I believe, also the cost grows as we need to store more data from month to month and will need updating. Maybe we should mention that. 3 months from now it will be more archival data and more cost 😸

File: docs/develop/node/archival/run-archival-node.md (hunk 2 above)
Comment (janewang) [reply docs/develop/node/archival/run-archival-node.md:36]: Thanks @chefsale. Would love to connect on neard and systemd. Thanks in advance on this!
@bowenwang1996 Will merge this PR as it stands and add archival instructions with @chefsale's help. Would like to make announcements in validators channels today and link to the docs. Thanks!

**Pull request merged**